### PR TITLE
Add `random` spinner

### PR DESCRIPTION
**Pull request by sbalay**
## Summary

I have been developing a CLI for a couple of days now and I think it is kinda fun to have a random spinner every time I use one if its commands.

I created a wrapper of cli-spinners in the CLI and developed this feature just for myself... but maybe other people also like it, so here you go.

Feel free to close the PR if you don't want this feature in the codebase :)

## Alternative implementation

Instead of having it as a function `cliSpinners.random()`, we could use something like `Object.defineProperty` to have it as an attribute `cliSpinners.random`

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Instead of having it as a function cliSpinners.random(), we could use something like Object.defineProperty to have it as an attribute cliSpinners.random

I usually prefer a method when the return value is not deterministic, but in this case, it could be nice that `.random` is just another spinner. It makes it *just work* in packages using this, like `ora`.

**Comment by sbalay** [comment]
@sindresorhus done! Not sure what's wrong with CI though 😕 

```
> xo && ava && tsd
Error: Cannot find module 'lodash/fp'
Require stack:
- /home/travis/build/sindresorhus/cli-spinners/node_modules/enhance-visitors/index.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/eslint-plugin-ava/rules/assertion-arguments.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/import-modules/index.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/eslint-plugin-ava/index.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/eslint/lib/config/plugins.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/eslint/lib/config.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/eslint/lib/cli-engine.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/eslint/lib/api.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/xo/index.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/xo/cli-main.js
- /home/travis/build/sindresorhus/cli-spinners/node_modules/xo/cli.js
Referenced from: 
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Module.require (internal/modules/cjs/loader.js:1026:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/home/travis/build/sindresorhus/cli-spinners/node_modules/enhance-visitors/index.js:3:11)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Module.require (internal/modules/cjs/loader.js:1026:19)

```

File: test.js [hunk 1]
```diff
@@ -1,8 +1,41 @@
...
+	}
+}
+
 test('main', t => {
 	t.is(typeof cliSpinners, 'object');
 	t.is(cliSpinners.dots.interval, 80);
 	t.true(Array.isArray(cliSpinners.dots.frames));
 });
+
+test('random getter', t => {
```
Comment (sbalay) [inline test.js:23]: Not sure what's the preferred style for tests in this repo. Let me know if you think we should structure these tests in a different way. (split the tests cases? move mock methods to a different file? etc)

**Comment by sindresorhus** [comment]
Can you mention in the API docs in the readme and index.d.ts that the `random` spinner is special and will return a random spinner?

File: test.js (hunk 1 above)
Comment (sindresorhus) [reply test.js:23]: It's good 👍🏻

**Comment by sindresorhus** [comment]
> done! Not sure what's wrong with CI though 😕

Don't worry about that. I'll deal with it.

**Comment by sbalay** [comment]
> Can you mention in the API docs in the readme and index.d.ts that the `random` spinner is special and will return a random spinner?

Done

File: index.d.ts [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
+		| "runner"
+		| "pong"
+		| "shark"
+		| "dqpb"
+		| "weather"
+		| "christmas"
+		| "grenade"
+		| "point"
+		| "layer"
+		| "betaWave";
```
Comment (sindresorhus) [inline index.d.ts:73]: Don't do unrelated changes.

File: readme.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ $ npm install cli-spinners
 ## Usage
 
 ```js
-const cliSpinners = require('cli-spinners');
+const cliSpinners = require("cli-spinners");
```
Comment (sindresorhus) [inline readme.md:25]: https://github.com/sindresorhus/cli-spinners/pull/47/files#r456775532

File: index.d.ts (hunk 2 above)
Comment (sbalay) [reply index.d.ts:73]: Yeah, just noticed them. I'll fix them in a minute!

File: readme.md (hunk 3 above)
Comment (sbalay) [reply readme.md]: Fixed

**Comment by sindresorhus** [comment]
Thanks :)
